<?xml version="1.0"?>
<launch>

    <arg name="target_nav_action_0" default="/turtlebot/move_base"/>
    <arg name="target_nav_action_1" default="/tb2/move_base"/>
<!--     <arg name="target_nav_action_2" default="tb3/move_base"/>
    <arg name="target_nav_action_3" default="tb4/move_base"/> -->
    <!--<arg name="target_nav_action_4" default="tb5/move_base"/>-->

    <arg name="target_path_0" default="/turtlebot/search_agent_manager/waypoint_plan"/>
    <arg name="target_path_1" default="/tb2/search_agent_manager/waypoint_plan"/>
<!--     <arg name="target_path_2" default="agent_2_path"/>
     <arg name="target_path_3" default="agent_3_path"/>  -->
    <!--<arg name="target_path_4" default="agent_4_path"/>-->

    <node name="path_follow_agent_0" pkg="search_service" type="path_follow_server" output="screen" args="$(arg target_nav_action_0) $(arg target_path_0)"/>
    <node name="path_follow_agent_1" pkg="search_service" type="path_follow_server" output="screen" args="$(arg target_nav_action_1) $(arg target_path_1)"/>
  <!--   <node name="path_follow_agent_2" pkg="search_service" type="path_follow_server" output="screen" args="$(arg target_nav_action_2) $(arg target_path_2)"/>
    <node name="path_follow_agent_3" pkg="search_service" type="path_follow_server" output="screen" args="$(arg target_nav_action_3) $(arg target_path_3)"/> -->
    <!--<node name="path_follow_agent_4" pkg="search_service" type="path_follow_server" output="screen" args="$(arg target_nav_action_4) $(arg target_path_4)"/>-->



</launch>
